scripts/table.js

```diff
@@ -1,12 +1,11 @@
-// TODO: Extract selection logic in separate child class to keep Single Responsibility Principle for SmartTable and avoid GOD object;
 class SmartTable {
     #tableContainerRef;
     #tableHeaderRef;
     #tableBodyRef;
     #colDef;
     #selectable;
 
-    constructor(selector, colDef, selectable = true) {
+    constructor(selector, colDef, selectable = false) {
```

Comment:
remove selectable from following places in SmartTable class, you should leave it only for SmartSelectableTable

Reply:
Fixed